libkbfs/bare_root_metadata.go

```diff
@@ -271,6 +271,12 @@ func (md *BareRootMetadataV2) DeepCopy(codec Codec) (BareRootMetadata, error) {
 	return &newMd, nil
 }
 
+// MakeSuccessor implements the ImmutableBareRootMetadata interface for BareRootMetadataV2.
+func (md *BareRootMetadataV2) MakeSuccessor(codec Codec) (BareRootMetadata, error) {
+	// MDv3 TODO: Make a v3 successor.
+	return md.DeepCopy(codec)
```

Comment:
I'm saying that this `MakeSuccessor` doesn't actually make a successor, since it doesn't bump the revision number, etc. I guess that'll be done in a future PR? It seems dangerous to leave this method around, though, since the caller might assume that it does the right thing.